index.bs

```diff
@@ -315,12 +375,22 @@ modifications to existing large files.
 <div algorithm>
 The <dfn method for=FileSystemFileHandle>createWritable(|options|)</dfn> method, when invoked, must run these steps:
 
-1. Issue: TODO: Check permissions.
-1. Let |stream| be the result of [=create a new FileSystemWritableFileStream|creating a new FileSystemWritableFileStream=]
-   for <b>[=this=]</b>'s [=FileSystemHandle/entry=].
-1. If |options|.{{FileSystemCreateWritableOptions/keepExistingData}} is `true`:
-   1. Set |stream|.[=[[buffer]]=] to a copy of <b>[=this=]</b>'s [=FileSystemHandle/entry=]'s [=file entry/binary data=].
-1. Return [=a promise resolved with=] |stream|.
+1. Let |result| be [=a new promise=].
+1. Run the following steps [=in parallel=]:
+  1. Run <b>[=this=]</b>'s [=FileSystemHandle/entry=]'s [=request permission steps=] given
+     «[&nbsp;"{{FileSystemHandlePermissionDescriptor/writable}}" → `true`&nbsp;]».
+     If that throws an exception, [=reject=] |result| with that exception and abort.
```

Comment:
Be consistent with the above, i.e. if the "catch" is part of the same step or not. Although it makes this line longer (and multi-part), I think I prefer this form, so maybe convert the above to be a single line? 

(No strong feelings here)